cli/azd/internal/appdetect/javascript.go

```diff
@@ -75,6 +78,15 @@ func (nd *javaScriptDetector) DetectProject(ctx context.Context, path string, en
 				}
 			}
 
+			for dep := range packagesJson.DevDependencies {
```

Comment:
Not the best handling right now for dependencies/devDependencies. Have some ideas here if we want to expand how general JS detection works, which includes SSR vs. static front-ends, but let's address this in the future if possible.